### PR TITLE
Don't suppress resolve telemetry or error messages

### LINKED ISSUE #235: App centric telemetry
* Make sure we're not suppressing resolve telemetry

### PR DESCRIPTION
**Pull request by alexweininger**
Fixes #235

### CODE REVIEW

**Review by nturinski — COMMENTED** [review]
Did we want to display the error though? The issue specifically mentions suppressing for telemetry, but not sure if the user needs/wants to know about these errors.

**Comment by alexweininger** [comment]
> Did we want to display the error though? The issue specifically mentions suppressing for telemetry, but not sure if the user needs/wants to know about these errors.

I was wondering the same thing. Was hoping you would have some epic wisdom on what we should do. I guess we shouldn't show it but I can't give a good reason why. Maybe we wrap it in try/catch and show a nicer error message?

File: src/extension.ts [hunk 1]
```diff
@@ -87,6 +89,8 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         registerApplicationResourceResolver('vscode-azureresourcegroups.wrapperResolver', wrapperResolver);
         registerApplicationResourceResolver('vscode-azureresourcegroups.installableAppResourceResolver', installableAppResourceResolver);
         registerApplicationResourceResolver('vscode-azureresourcegroups.shallowResourceResolver', shallowResourceResolver);
+
+        await vscode.commands.executeCommand('setContext', 'azure-account.signedIn', await ext.rootAccountTreeItem.getIsLoggedIn());
```
Comment (nturinski) [inline src/extension.ts:93]: Is this related to the walkthrough?

**Comment by nturinski** [comment]
> > Did we want to display the error though? The issue specifically mentions suppressing for telemetry, but not sure if the user needs/wants to know about these errors.
> 
> I was wondering the same thing. Was hoping you would have some epic wisdom on what we should do. I guess we shouldn't show it but I can't give a good reason why. Maybe we wrap it in try/catch and show a nicer error message?

I'm actually not even sure what it looks like when it does error.  I'm worried that one error might prevent the whole group from being displayed.  I'll try forcing an error for the resolve-- might have a better insight after that.  But I think it's safe to suppress for now.

File: src/extension.ts (hunk 1 above)
Comment (alexweininger) [reply src/extension.ts:93]: Damn, how did that get there

**Review by nturinski — APPROVED** [review]
